learn/sherif/main_test/src/main/java/frc/robot/commands/Intake.java

```diff
@@ -16,40 +16,40 @@
     public static final int THROAT_SPEED = 0;
     public static final int ELEVATOR_SPEED = 1;
     public static final int FEEDER_SPEED = 2;
-    public double m_speeds[] = {1.0, -1.0, 1.0};
+    public double[] speeds = {1.0, -1.0, 1.0};
     public boolean elevatorLeft = false;
     public boolean elevatorRight = false;
     public boolean throat = false;
     public boolean feeder = false;
 
-    private final IntakeControl m_intakeControl;
-    private final TestableJoystick m_joystick;
-    private Slider m_sliders[] = {null, null, null};
+    private final IntakeControl intakeControl;
+    private final TestableJoystick joystick;
+    private final Slider[] sliders = {null, null, null};
```

Comment:
This could also be achieved with `new Slider[3];` but I don't know if there's any significant difference between the two methods.